backend/src/openarchiefbeheer/destruction/tests/endpoints/test_co_reviews.py

```diff
@@ -0,0 +1,147 @@
+from django.urls import reverse
+
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from openarchiefbeheer.accounts.tests.factories import UserFactory
+from openarchiefbeheer.destruction.constants import ListRole, ListStatus
+from openarchiefbeheer.destruction.tests.factories import (
+    DestructionListAssigneeFactory,
+    DestructionListCoReviewFactory,
+    DestructionListFactory,
+)
+
+
+class CoReviewsViewSetTest(APITestCase):
+    def test_list_not_logged_in(self):
+        url = reverse("api:destruction-list-co-reviews-list")
+
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_list(self):
+        user = UserFactory()
+
+        self.client.force_login(user)
+        url = reverse("api:destruction-list-co-reviews-list")
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_list_filter_destruction_list__uuid_not_logged_in(self):
+        destruction_list = DestructionListFactory.create(
+            status=ListStatus.ready_to_review
+        )
+
+        url = reverse("api:destruction-list-co-reviews-list")
+        response = self.client.get(
+            f"{url}?destruction_list__uuid={destruction_list.uuid}"
+        )
```

Comment:
In the other tests I try to use furl to build URLs with query params, but not crucial :smile: